pages/dao/[symbol]/proposal/[pk]/explore.tsx

```diff
@@ -53,19 +78,47 @@ export default function Explore() {
             <h1 className="mr-2">{proposal?.account.name}</h1>
             <ProposalStateBadge proposal={proposal.account} />
           </div>
-          <h3 className="mb-4 mt-16">Top Voters</h3>
+          <div className="mb-4 mt-16 flex justify-between">
+            <h3 className="">Top Voters</h3>
+            <div
+              className={`${
+                isNftMode ? 'visible' : 'hidden'
+              } flex items-center`}
+            >
+              <p className="mb-0 mr-1 text-fgd-3">Show NFTs</p>
+              <Switch
+                checked={showNfts}
+                onChange={() => {
+                  toggleShowNfts()
+                }}
+              />
+            </div>
+          </div>
           <div
             className="grid gap-4 grid-cols-1 items-center lg:grid-cols-2"
             onMouseLeave={() => setHighlighted(undefined)}
           >
-            <ProposalTopVotersList
-              className="h-[500px]"
-              data={records}
-              endpoint={endpoint}
-              isMulti={isMulti}
-              highlighted={highlighted}
-              onHighlight={setHighlighted}
-            />
+            <div className="flex flex-col gap-5 h-[500px]">
+              <ProposalTopVotersList
+                className={showNfts ? 'h-[275px]' : 'h-[500px]'}
+                data={records}
+                endpoint={endpoint}
+                isMulti={isMulti}
+                highlighted={highlighted}
+                onHighlight={setHighlighted}
+              />
+              <ProposalTopVotersNftChart
```

Comment:
a comment here explaining the difference between this and `ProposalTopVotersList` maybe would be nice